variables/_svgs.pcss

```diff
@@ -70,13 +82,19 @@
 	}
 }
 
+/* -----------------------------------------------------------------------------
+ * SVG - Day
+ * ----------------------------------------------------------------------------- */
+
+@svg-load day url('../../../images/icons/day.svg') {}
```

Comment:
I think it'd be a nice idea to manage the color/fill via CSS, in case these are updated later by other devs.